schemas/context/experienceevent.example.4.json

```diff
@@ -143,7 +168,9 @@
       },
       "xdm:pauseTime": {
         "xdm:value": 0
-      }
+      },
+      "xdm:idp": "MTV",
+      "xdm:mediaAuth": "true"
```

Comment:
again, bad boolean

Reply:
This is not a boolean variable. If usually has a true/false value, but the clients can send any text value. Will add some extra examples with different values to make it more clear.